packages/manager/apps/web-paas/package.json

```
@@ -18,6 +18,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-web-paas-app' --include-dependencies -- npm run dev:watch --if-present"
   },
   "dependencies": {
+    "@ovh-ux/manager-billing": "^1.0.5 || ^2.0.0",
```

```suggestion
    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```

this module just have been renamed